### PR TITLE
Fix deqp tests to abide to the clear/clearBuffer function/buffer matc…

### PR DESCRIPTION
**Pull request by zhenyao**
…hing rule.

### CODE REVIEW

File: sdk/tests/deqp/functional/gles3/es3fFboColorbufferTests.js [hunk 1]
```diff
@@ -226,7 +226,6 @@ es3fFboColorbufferTests.FboColorClearCase.prototype.render = function(dst) {
             var color = es3fFboColorbufferTests.randomVector(rnd, fmtInfo.valueMin, fmtInfo.valueMax);
 
             ctx.scissor(x, y, w, h);
-            ctx.clearBufferfv(gl.COLOR, 0, color);
```
Comment (zhenyao) [inline sdk/tests/deqp/functional/gles3/es3fFboColorbufferTests.js]: c++ doesn't have this call: https://cs.chromium.org/chromium/src/third_party/deqp/src/modules/gles3/functional/es3fFboColorbufferTests.cpp?rcl=0&l=138

File: sdk/tests/deqp/functional/gles3/es3fFboInvalidateTests.js [hunk 2]
```diff
@@ -843,8 +843,8 @@ es3fFboInvalidateTests.InvalidateSubFboUnbindReadCase.prototype.render = functio
 
     this.checkFramebufferStatus(gl.FRAMEBUFFER);
 
-    ctx.clearColor(0, 0, 0, 1);
-    ctx.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT | gl.STENCIL_BUFFER_BIT);
+    this.clearColorBuffer(colorFmt, [0.0, 0.0, 0.0, 1.0]);
+    ctx.clear(gl.DEPTH_BUFFER_BIT | gl.STENCIL_BUFFER_BIT);
```
Comment (zhenyao) [inline sdk/tests/deqp/functional/gles3/es3fFboInvalidateTests.js]: https://cs.chromium.org/chromium/src/third_party/deqp/src/modules/gles3/functional/es3fFboInvalidateTests.cpp?rcl=0&l=985

**Comment by zhenyao** [comment]
This needs to land and be rolled into chromium before we can land https://codereview.chromium.org/2152213002/

@Kangz @kenrussell Please take a look.

**Comment by Kangz** [comment]
Nice fix, LGTM!

**Comment by kenrussell** [comment]
Very nice! LGTM after the fact.

**Pull request merged**